staff/emma-cowdery/cv/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Emma Cowdery CV</title>
+</head>
+<body>
+    <header>
+        <h1 id="top">CV EMMA COWDERY</h1>
+    </header>
+    <br>
+    <table>
+        <tr>
+            <td><a href="#Basic information">Basic information</a></td>
+            <td><a href="#Education">Education</a></td>
+            <td><a href="#Other certificates">Other certificates</a></td>
+            <td><a href="#Languages and program knowledge">Languages and program knowledge</a></td>
+            <td><a href="#Employment">Employment</a></td>
+            <td><a href="#Additional information">Additional information</a></td>
+        </tr>
+    </table>
+    <img src="nai.png"/>
+    <section>
+    <h2 id="Basic information">Basic information</h2>
+    <ul>
+        <li><i>Carrer Sant Pere Més Baix, 51, 4/2, 08003, Barcelona</i></li>
+        <li><i>674 33 71 51</i></li>
+        <li><i>Date of birth: 19/06/1999</i></li>
+        <li><i>ercowdery@gmail.com</i></li>
+    </ul>
+    </section>
+    <br>
+    <section>
+    <h2 id="Education">Education</h2>
+    <p>
+        <ul>
+            <li><b>Degree in Nanoscience and Nanotechnology</b><br>Autonomous University of (initiated in 2017)</li>
+            <br>
+            <li><b>High school (graduating with honours)</b><br>INS Verdaguer, Barcelona (2015-2017)</li>
+        </ul>
+    </p>
+    </section>
+    <br>
+    <section>
+    <h2 id="Other certificates">Other certificates</h2>
+    <p>
+        <ul>
+            <li><b>Advanced Certificate in English</b><br>International House, Barcelona (2017)</li>
+            <br>
+            <li><b>Curso Técnico Colaborador (sailing instructor certificate)</b><br>Catalan Federation of Sailing, Barcelona (2016)</li>
+        </ul>
+    </p>
+    </section>
+    <br>
+    <section>
+    <h2 id="Languages and program knowledge">Languages and program knowledge</h2>
+    <p>
+        <ul>
+            <li><b>Catalan</b><br>Native level</li>
+            <br>
+            <li><b>Spanish</b><br>Native level</li>
+            <br>
+            <li><b>English</b><br>Native level</li>
+            <br>
+            <li><b>Microsoft Office</b><br>Advanced understanding</li>
+        </ul>
+    </p>
+    </section>
+    <br>
+    <section>
+    <h2 id="Employment">Employment</h2>
+    <p>
+        <ul>
+            <li><b>Outdoor activity coordinator</b><br>Municipal centre of sailing, Barcelona (2016-2018)</li>
+            <br>
+            <li><b>Private tutoring in English, Mathematics, Physics, Chemistry and Biology</b><br>High school and middle school level students (2015-2018)</li>
+        </ul>
+    </p>
+    </section>
+    <br>
+    <section>
+    <h2 id="Additional information">Additional information</h2>
+    <p>
```

Comment:
once again, you are using `p` just because of a visual thing, but has accurate no semantic here, because you are presenting a list at the end. remove p.